backend/controllers/userController.js

```diff
@@ -56,8 +57,44 @@ const deleteUser = async (req, res) => {
   }
 };
 
+const updatePassword = async (req, res) => {
+  try {
+    const { current_password, new_password, re_new_password } = req.body;
+    console.log(new_password.length + " password");
```

Comment:
Let's remove unnecessary console logs like this one

Reply:
Done 👍